packages/web-workers/src/controllers/health-monitor-orchestration-controller.ts

```diff
@@ -70,16 +70,26 @@ export class HealthMonitorOrchestrationController extends WebController {
             yield* orchestrationSteps.invokeHealthCheckRestApi();
 
             const scanId = yield* orchestrationSteps.invokeSubmitScanRequestRestApi(availabilityTestConfig.urlToScan);
+            const consolidatedScanId = yield* orchestrationSteps.invokeSubmitConsolidatedScanRequestRestApi(
+                availabilityTestConfig.urlToScan,
+                availabilityTestConfig.consolidatedReportId,
```

Comment:
This doesn't necessarily need to be done in this PR, but it might be a good idea to append the current releaseId to the consolidatedReportId here. If we use the same consolidatedReportId for all E2E tests across releases, there will only ever be one consolidated report generated. That might lead to false test passes if consolidated report generation breaks on a particular release, but we can still fetch the previously generated report from storage.

Reply:
I like this idea and think we can do that in a separate PR if you're fine with that!